damageproto/plan.sh

```diff
@@ -0,0 +1,19 @@
+pkg_name=damageproto
+pkg_origin=core
+pkg_version=1.2.1
+pkg_description="X11 Damage C header files"
+pkg_upstream_url="https://www.x.org/"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source="https://www.x.org/releases/individual/proto/${pkg_name}-${pkg_version}.tar.bz2"
+pkg_shasum=5c7c112e9b9ea8a9d5b019e5f17d481ae20f766cb7a4648360e7c1b46fc9fc5b
+pkg_deps=(
+)
```

Comment:
Interesting.  As a follow up action item we might want to figure out how many of these are just wrong:

```
> grep 'pkg_dep' *proto/plan.sh
fixesproto/plan.sh:pkg_deps=(core/glibc)
inputproto/plan.sh:pkg_deps=(core/glibc)
recordproto/plan.sh:pkg_deps=(core/glibc)
renderproto/plan.sh:pkg_deps=(core/glibc)
xcb-proto/plan.sh:pkg_deps=(core/glibc)
xextproto/plan.sh:pkg_deps=(core/glibc)
```

My guess is most since I think these proto packages are just header files.

Reply:
I'll open an issue with your comment so we don't forget :)